### PR TITLE
Webpage updates

### PR DESCRIPTION
**Pull request by CarolineHana**
styling of blog design layout for main blog page with columns, navigation bar, and about me post. Translated page also included.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,20 +2,18 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="indexstyle.css">
```
Comment (ciddu) [inline portfolio/src/main/webapp/index.html]: prefer using _ to separate two words in filenames. index_style

File: portfolio/src/main/webapp/mainARB.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (ciddu) [inline portfolio/src/main/webapp/mainARB.html:1]: can you rename it to main_ARB.html

File: portfolio/src/main/webapp/mainENG.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (ciddu) [inline portfolio/src/main/webapp/mainENG.html:1]: similar renaming here.

File: portfolio/src/main/webapp/mainENG.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="stylesheet" href="mainstyle.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+	<div class ="blog">
+	<div class = "row">
+     	<div class= "column left">
+			<img class= "profile" src="/Images/profile2.jpg" alt="Caroline" width="300" height="400">
+            <p> Hello Readers, <p>
+        	<p> Im Caroline Hana and this is my Blog! Im a 18 year old college student currently living in NYC. I currently attend school at Princeton University, but I say that loosely because we’re all in online and home right now. This blog is part of a project series within the Google Student Training in Engineering Program. </p>
```
Comment (ciddu) [inline portfolio/src/main/webapp/mainENG.html:15]: Can you follow 80 character line limit. It's difficult to scroll to the end.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -2,20 +2,18 @@
...
-     <img src="/Images/profile2.jpg" alt="Caroline" width="300" height="400">
-     <img src="/Images/avatar.jpg" alt="Caroline" width="300" height="300">
-      <button onclick="addRandomFunFacts()">Press Me!</button>
-      
-      <div id="facts-container"></div>
+    <div class="DefaultPageImage"></div>
+    <div class ="MainPageText">
+      <h1>Welcome to my blog!</h1>
+      <h2> Select your preferred language below: </h2>
+      <button onclick="EnglishPage()">English</button>
```
Comment (ciddu) [inline portfolio/src/main/webapp/index.html]: You could also try <a href="/main_ENG.html"> English </a>. You won't need any JavaScript in that case.

**Review by ciddu — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (CarolineHana) [reply portfolio/src/main/webapp/index.html]: I agree. The code is much clearer and simple by removing the javascript function.

File: portfolio/src/main/webapp/mainENG.html (hunk 4 above)
Comment (CarolineHana) [reply portfolio/src/main/webapp/mainENG.html:15]: Overlooked that sorry. Its changed.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (ciddu) [inline portfolio/src/main/webapp/index.html]: index_style

File: portfolio/src/main/webapp/indexstyle.css [hunk 6]
```diff
@@ -0,0 +1,32 @@
+body, html {
```
Comment (ciddu) [inline portfolio/src/main/webapp/indexstyle.css]: delete this file

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (ciddu) [reply portfolio/src/main/webapp/index.html]: It looks like there are two versions of the CSS and HTML files. One with _ and one without. Can you remove unwanted ones.

**Review by ciddu — CHANGES_REQUESTED** [review]

**Review by ciddu — APPROVED** [review]

**Pull request merged**